doc/part-hydras.tex

```diff
@@ -2512,7 +2512,11 @@ \section{A notation for finite ordinals}
 \href{https://coq.inria.fr/distrib/current/stdlib/Coq.Logic.Eqdep_dec.html}{Coq.Logic.Eqdep\_dec}.
 
 \index{Coq}{Unicity of equality proofs}
+<<<<<<< HEAD
+ 
+=======
 
+>>>>>>> master
```

Comment:
```suggestion

```
I think this is a remnant of the merge that should be fixed. Otherwise, I think everything is good here, so unless anyone has objections I will merge the PR tomorrow.